apps/docs/content/guides/auth/debugging/error-codes.mdx

```diff
@@ -1,3 +1,12 @@
+---
+id: 'auth-error-codes'
+title: 'Error Codes'
+description: 'Supabase Auth Error Codes'
+subtitle: 'Learn about the Auth error codes and how to resolve them'
+---
+
+## Auth Error Codes
+
 Supabase Auth can throw or return various errors when using the API. All errors originating from the `supabase.auth` namespace of the client library will be wrapped by the `AuthError` class.
```

Comment:
One problem with this brought up by Tyler (and why he abstracted the AuthErrorCodes table in the first place) is that the intro is too JavaScript-centric (the listed classes are all JavaScript-based, for example). We might need to have tabs for the different language details

e.g., this is what Tyler wrote for [Dart](https://docs-git-j0-moveerrorcodes-supabase.vercel.app/docs/reference/dart/auth-error-codes)